ui/src/AggStats/index.js

```diff
@@ -1,26 +1,13 @@
 import "whatwg-fetch";
 import React, { Component } from "react";
-import {
-  Container,
-  Row,
-  Col,
-  Button,
-  Card,
-  CardText,
-  CardTitle
-} from "reactstrap";
+import { Container, Row, Col, Button, Card, CardTitle } from "reactstrap";
 
 export default class AggStats extends Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      success: 0,
-      error: 0,
-      mean_resp_time: 0
-    };
-    this.handleFetch = this.handleFetch.bind(this);
```

Comment:
binding is not required anymore because of fancy babel config that enable new feature?

Reply:
using arrow functions solves the issue, they keep `this` binded.

Reply:
ah, got it